Application/mobile/app_code/lib/pages/rent/confirm/confirm_rent_state.dart

```diff
@@ -0,0 +1,156 @@
+  void initState() {
+    super.initState();
+    hours = widget.hours;
+    data = widget.data;
+  }
+
+  String pluralHours() {
+    if (hours > 1) {
+      return "s";
+    } else {
+      return "";
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      resizeToAvoidBottomInset: false,
+      backgroundColor: context.select((ThemeProvider themeProvider) =>
+          themeProvider.currentTheme.colorScheme.background),
+      appBar: MyAppBar(
+        curveColor: context.select((ThemeProvider themeProvider) =>
+            themeProvider.currentTheme.secondaryHeaderColor),
+        showBackButton: false,
+        showLogo: true,
+        showBurgerMenu: false,
+      ),
+      body: SingleChildScrollView(
+        child: Container(
+          margin: const EdgeInsets.symmetric(horizontal: 16, vertical: 32),
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.start,
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              Center(
+                child: Column(
+                  children: [
+                    Text(
+                      'Confirmation de location:',
+                      style: TextStyle(
+                        fontSize: 28,
+                        fontWeight: FontWeight.bold,
+                        decoration: TextDecoration.underline,
+                        color: context.select((ThemeProvider themeProvider) =>
+                            themeProvider.currentTheme.secondaryHeaderColor),
+                      ),
+                    ),
+                    Text(
+                      data.name,
+                      style: TextStyle(
+                        fontSize: 26,
+                        fontWeight: FontWeight.bold,
+                        color: context.select((ThemeProvider themeProvider) =>
+                            themeProvider.currentTheme.primaryColor),
+                      ),
+                    ),
+                  ],
+                ),
+              ),
+              const SizedBox(height: 32),
+              Text(
+                'Résumé:',
+                style: TextStyle(
+                  fontSize: 22,
+                  fontWeight: FontWeight.bold,
+                  color: context.select((ThemeProvider themeProvider) =>
+                      themeProvider.currentTheme.secondaryHeaderColor),
+                ),
+              ),
+              const SizedBox(height: 8),
+              Text(
+                '- Prix par heure: ${data.price} euros',
+                style: TextStyle(
+                  fontSize: 22,
+                  fontWeight: FontWeight.bold,
+                  color: context.select((ThemeProvider themeProvider) =>
+                      themeProvider.currentTheme.primaryColor),
+                ),
+              ),
+              const SizedBox(height: 8),
+              Text(
+                '- Nombre d\'heure${pluralHours()}: $hours',
```

Comment:
Don't need to create a function for it, ${hours > 1 ? 's' : ''} should be enough